### PR TITLE
PLAT-126336: Added UI tests for DatePicker

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - Added ui tests for DatePicker

### Links
[//]: # (Related issues, references)
PLAT-126336

### Comments

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]

File: tests/ui/specs/DatePicker/Agate-DatePicker-Page.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+'use strict';
```
Comment (jeonghee27) [inline tests/ui/specs/DatePicker/Agate-DatePicker-Page.js:1]: You don't need to use'-' for Page file name.
In the PLAT-126330, there was not.
Agate-PanelPage.js is Better.
In fact, I don't think it's good to prefix of "Agate-".
"-Spec" and "-View" is mandatory to filter on ui-test-utils. but Page is not. just use camelcase.

Comment (daniel-stoian-lgp) [reply tests/ui/specs/DatePicker/Agate-DatePicker-Page.js:1]: I will change. thanks!

**Review by MikyungKim — COMMENTED** [review]
@daniel-stoian-lgp , have you run the ui test with this branch?
I got 
```
[0-0] Error in "DatePicker LTR locale default should have month-day-year order"
Can't call $ on element with selector "#datePickerDefault" because element wasn't found
```
Have you seen?

**Comment by daniel-stoian-lgp** [comment]
> @daniel-stoian-lgp , have you run the ui test with this branch?
> I got
> 
> ```
> [0-0] Error in "DatePicker LTR locale default should have month-day-year order"
> Can't call $ on element with selector "#datePickerDefault" because element wasn't found
> ```
> 
> Have you seen?

I did run it multiple times. It never failed with this message. I will check again

**Comment by MikyungKim** [comment]
Maybe it's my machine problem. let me check 😕

**Comment by daniel-stoian-lgp** [comment]
> Maybe it's my machine problem. let me check 😕

I pushed a minor fix, but it wasn't causing any issue anyway. My most recent output

```
[chrome 86.0.4240.183 Windows NT #0-0] Running: chrome (v86.0.4240.183) on Windows NT
[chrome 86.0.4240.183 Windows NT #0-0] Session ID: e2a47a0c85f6dac53750aac30288a23f
[chrome 86.0.4240.183 Windows NT #0-0]
[chrome 86.0.4240.183 Windows NT #0-0] DatePicker
[chrome 86.0.4240.183 Windows NT #0-0]     LTR locale
[chrome 86.0.4240.183 Windows NT #0-0]         default
[chrome 86.0.4240.183 Windows NT #0-0]            ✓ should have correct title
[chrome 86.0.4240.183 Windows NT #0-0]            ✓ should have month-day-year order
[chrome 86.0.4240.183 Windows NT #0-0]
[chrome 86.0.4240.183 Windows NT #0-0]             5-way
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should increase the month when incrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should decrease the month when decrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should increase the day when incrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should decrease the day when decrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should increase the year when incrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should decrease the year when decrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]
[chrome 86.0.4240.183 Windows NT #0-0]             pointer
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should increase the month when incrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should decrease the month when decrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should increase the day when incrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should decrease the day when decrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should increase the year when incrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should decrease the year when decrementing the picker
[chrome 86.0.4240.183 Windows NT #0-0]
[chrome 86.0.4240.183 Windows NT #0-0]         with 'defaultValue'
[chrome 86.0.4240.183 Windows NT #0-0]             5-way
[chrome 86.0.4240.183 Windows NT #0-0]                ✓ should not update on select
[chrome 86.0.4240.183 Windows NT #0-0]
[chrome 86.0.4240.183 Windows NT #0-0]         disabled
[chrome 86.0.4240.183 Windows NT #0-0]            ✓ should focus the disabled month picker
[chrome 86.0.4240.183 Windows NT #0-0]            ✓ should not increase the day when incrementing disabled picker
[chrome 86.0.4240.183 Windows NT #0-0]            ✓ should not decrease the day when decrementing disabled picker
[chrome 86.0.4240.183 Windows NT #0-0]
[chrome 86.0.4240.183 Windows NT #0-0]         disabled with 'defaultValue'
[chrome 86.0.4240.183 Windows NT #0-0]            ✓ should display default date
[chrome 86.0.4240.183 Windows NT #0-0]            ✓ should not update 'defaultValue' on decrementing disabled picker
[chrome 86.0.4240.183 Windows NT #0-0]            ✓ should not update 'defaultValue' on incrementing disabled picker
[chrome 86.0.4240.183 Windows NT #0-0]
[chrome 86.0.4240.183 Windows NT #0-0]     RTL locale
[chrome 86.0.4240.183 Windows NT #0-0]        ✓ should focus rightmost picker (day) when selected
[chrome 86.0.4240.183 Windows NT #0-0]        ✓ should have day-month-year order
[chrome 86.0.4240.183 Windows NT #0-0]
[chrome 86.0.4240.183 Windows NT #0-0] 23 passing (26.1s)
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**